Micro-dep-acct-eft-purge/REPWRITERSPECS/BANNO.MICRO.BATCH.FM.V1.POW

```diff
@@ -0,0 +1,81 @@
+[*
+**  PowerOn Name:       BANNO.MICRO.BATCH.FM.V1.POW
+**  Letterfile Name:    N/A
+**
+**  Copyright 2021 Jack Henry and Associates
+**
+**  This Banno PowerOn is used in Episys via Batch Control.
+**  It will produce a report of EFT records to delete.
+**  The CU must run a Miscellaneous Process to actually FM
+**  the account. 
+**  The user enters the micro account number and the number
+**  of days to keep. It selects the EFT record as long as 
+**  it is a single frequency, is expired and was processed. 
+**  
+**  The CU must run a Miscellaneous Process to actually FM
+**  the account.
+**
+**  Any assistance with using this PowerOn in Episys will  
+**  be provided by Symitar support.
+**
+**  For more information check
+**  https://github.com/Banno/banno-powerons
+**
+**  Banno is not responsible for any modifications to this file
+**  made by unauthorized personnel.
+**
+**  Modification History:
+**    Ver. 1.0.0  04/15/2021 CM Simms - Original Programming
+**
+**  DO NOT MODIFY THIS FILE UNLESS YOU KNOW WHAT YOU'RE DOING!
+*]
+ 
+TARGET=EFT 
+
+DEFINE
+ DAYSTOKEEP=NUMBER
+ MICROACCOUNT=CHARACTER(10)
+ COUNT=NUMBER
+
+ TEMPCHAR=CHARACTER
+ MAXCHARS=NUMBER
+
+END
+
+SETUP
+ DAYSTOKEEP=NUMBERREAD("ENTER HOW MANY DAYS TO GO BACK")
+ TEMPCHAR=CHARACTERREAD("ENTER THE MICRO ACCOUNT NUMBER")
```

Comment:
There's no check to see if the user inadvertently entered over 10 characters.  Since the MICROACCOUNT variable is limited to 10 characters, the program would only capture the first 10 and assume it's valid.

Reply:
I don't think this is needed. I tried to put it in and various other checks, but when the account number is invalid, it does not find anything and ignores the print division. There was no way for me to tell the user what happened. I will make it a point in the readme to stress checking the output before posting.